Changes for issues in IDE integration (#1820)
aws-amplify/docs: docs/lib/project-setup/fragments/android/create-application/20_gradle.md

apply plugin: 'com.amplifyframework.amplifytools'
```

> - Add the line `classpath 'com.amplifyframework:amplify-tools-gradle-plugin:1.0.1'` within the `dependencies` blocks in both `buildscript` and `allprojects` blocks
> - Add the line `mavenCentral()` within the `repositories` block which is within the `allprojects` block

Reviewer: This doesn't match what we have, above -- I think it should read:
```suggestion
- Add the line `classpath 'com.amplifyframework:amplify-tools-gradle-plugin:1.0.1'` to the `dependencies` block inside `buildscript`.
- Add `mavenCentral()` to each of the `repositories` blocks under `buildscript` and `allprojects`.
```

Author: Ah yeah, I blew it. Will fix.
